sample/F8/speakers

```diff
@@ -0,0 +1,2631 @@
+      "country":"Not available",
+      "short_biography":"",
+      "website":"",
+      "twitter":"",
+      "facebook":"",
+      "github":"",
+      "linkedin":"",
+      "featured":false,
+      "position":"Executive Producer, Oculus",
+      "city":"",
+      "gender":"female",
+      "heard_from":"",
+      "icon":"",
+      "small":"",
+      "speaking_experience":"",
+      "sponsorship_required":"",
+      "thumbnail":""
+   },
+   {
+      "id":19,
+      "name":"Julia Sourikoff",
+      "email":"",
+      "mobile":"",
+      "photo":"",
+      "sessions":[
+         {
+            "id":14,
+            "title":"Bringing VR Experiences to Everyone: Scripting, Producing and Planning"
+         }
+      ],
+      "country":"Not available",
+      "short_biography":"",
+      "website":"",
+      "twitter":"",
+      "facebook":"",
+      "github":"",
+      "linkedin":"",
+      "featured":false,
+      "position":"Head of VR, Tool North America",
+      "city":"",
+      "gender":"female",
+      "heard_from":"",
+      "icon":"",
+      "small":"",
+      "speaking_experience":"",
+      "sponsorship_required":"",
+      "thumbnail":""
+   },
+   {
+      "id":20,
+      "name":"Christina Heller",
+      "email":"",
+      "mobile":"",
+      "photo":"",
+      "organisation":"Facebook",
+      "sessions":[
+         {
+            "id":14,
+            "title":"Bringing VR Experiences to Everyone: Scripting, Producing and Planning"
+         }
+      ],
+      "country":"Not available",
+      "short_biography":"",
+      "website":"",
+      "twitter":"",
+      "facebook":"",
+      "github":"",
+      "linkedin":"",
+      "featured":false,
+      "position":"CEO, VR Playhouse",
+      "city":"",
+      "gender":"male",
+      "heard_from":"",
+      "icon":"",
+      "small":"",
+      "speaking_experience":"",
+      "sponsorship_required":"",
+      "thumbnail":""
+   },
+   {
+      "id":21,
+      "name":"Ryan Horrigan",
+      "email":"",
+      "mobile":"",
+      "photo":"",
+      "organisation":"Facebook",
+      "sessions":[
+         {
+            "id":14,
+            "title":"Bringing VR Experiences to Everyone: Scripting, Producing and Planning"
+         }
+      ],
+      "country":"Not available",
+      "short_biography":"",
+      "website":"",
+      "twitter":"",
+      "facebook":"",
+      "github":"",
+      "linkedin":"",
+      "featured":false,
+      "position":"CCO, Felix and Paul Studios",
+      "city":"",
+      "gender":"male",
+      "heard_from":"",
+      "icon":"",
+      "small":"",
+      "speaking_experience":"",
+      "sponsorship_required":"",
+      "thumbnail":""
+   },
+   {
+      "id":22,
+      "name":"Tejal Parekh",
+      "email":"",
+      "mobile":"",
+      "photo":"",
+      "organisation":"Facebook",
+      "sessions":[
+         {
+            "id":15,
+            "title":"Building Native Games for Facebook Gameroom"
+         }
+      ],
+      "country":"Not available",
```

Comment:
I see "Not available" in countries but other fields like city, speaking experience and short biography are empty. Should we put some indicator in those too or leave them null?